LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
-                           Version 2.0, January 2004
```

Comment:
Did this licence file get moved somewhere else? Or got deleted by accident?

Reply:
Deleted by accident. Re-added in next commit 👍